MediaGallerySynchronization/Model/SelectByBatchesGenerator.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\MediaGallerySynchronization\Model;
+
+use Magento\Framework\App\ResourceConnection;
+use Psr\Log\LoggerInterface;
+use Magento\MediaGallerySynchronizationApi\Model\FetchBatchesInterface;
+
+/**
+ * Select data from database by provided batch size
+ */
+class SelectByBatchesGenerator implements FetchBatchesInterface
```

Comment:
Please update all properties naming accordingly
```suggestion
class FetchBatches implements FetchBatchesInterface
```